src/modules/governance/proposal/VotersListItem.tsx

```diff
@@ -47,7 +47,15 @@ export const VotersListItem = ({ voter }: VotersListItemProps): JSX.Element | nu
   // Zero decimals for 1000<=n<Infinity.
   const displayVotingPower = proposalVotingPower / 10 ** 18;
   const displayVotingPowerDecimals =
-    displayVotingPower < 1 ? 4 : displayVotingPower < 10 ? 3 : displayVotingPower < 1000 ? 2 : 0;
+    displayVotingPower < 1
+      ? 4
+      : displayVotingPower < 10
+      ? 3
+      : displayVotingPower < 1000 || displayVotingPower > 1000000
+      ? 2
+      : displayVotingPower > 100000
+      ? 1
+      : 0;
```

Comment:
👍 , so we'll have 1 decimal place for greater than 100,000.  Is this purely to initiate the formatting to round down, or a choice for better readability/design?

Reply:
Just for readability